EIPS/eip-multi_party_contract.md

```diff
@@ -0,0 +1,320 @@
+---
```

Comment:
You will need to add a `discussions-to` link for your EIP to be merged as a draft.

Reply:
Added the discussions-to